tests/DataCollector/PheanstalkDataCollectorTest.php

```diff
@@ -4,45 +4,18 @@
 
 use Leezy\PheanstalkBundle\DataCollector\PheanstalkDataCollector;
 use Leezy\PheanstalkBundle\PheanstalkLocator;
-use Pheanstalk\Connection;
-use Pheanstalk\PheanstalkInterface;
+use Pheanstalk\Contract\PheanstalkInterface;
+use PHPUnit\Framework\TestCase;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-class PheanstalkDataCollectorTest extends \PHPUnit_Framework_TestCase
+class PheanstalkDataCollectorTest extends TestCase
 {
     public function testCollect()
     {
-        $pheanstalkConnection = $this->getMockBuilder(Connection::class)
-            ->disableOriginalConstructor()
-            ->getMock();
-
-        $pheanstalkConnection
-            ->expects($this->any())
-            ->method('getHost')
-            ->will($this->returnValue('127.0.0.1'));
-
-        $pheanstalkConnection
-            ->expects($this->any())
-            ->method('getPort')
-            ->will($this->returnValue('11130'));
-
-        $pheanstalkConnection
-            ->expects($this->any())
-            ->method('getConnectTimeout')
-            ->will($this->returnValue(60));
-
-        $pheanstalkConnection
-            ->expects($this->any())
-            ->method('isServiceListening')
-            ->will($this->returnValue(false));
-
         $pheanstalkA = $this->getMockForAbstractClass(PheanstalkInterface::class);
```

Comment:
This test currently fails because `stats()` is not mocked which leads to an undefined index access at https://github.com/armetiz/LeezyPheanstalkBundle/blob/8858b2d24e90107f9254a1eaf7d3e050c938cbc5/src/DataCollector/PheanstalkDataCollector.php#L79